src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -60,6 +60,14 @@ using std::auto_ptr;
 
 /* ****************************************************************************
 *
+* OtisTreatFunction - callback signature for treatOntimeintervalSubscriptions
+*/
+typedef void (*OtisTreatFunction)(std::string tenant, BSONObj* bobjP);
```

Comment:
I just don't have it in my blood.
Never liked it.
I saw somewhere that C++ references should be used only when 'const XXX&' - when the object passed in is 'read-only'.
Let's look it up.